README.md

```diff
@@ -163,3 +163,23 @@ defined by a composition of a translation and a linear transformation. An
 (and controllable) if you construct it from a composition of a linear map
 and a translation, e.g. `Translation(v) ∘ LinearMap(v)` (or any combination of
 `LinearMap`, `Translation` and `AffineMap`).
+
+#### Perspective transformations
+
+The perspective transformation maps real-space coordinates to those on a virtual
+"screen" of one lesser dimension. For instance, this process is used to render
+3D scenes to 2D images in computer generated graphics and games. It is an ideal
+model of how a pinhole camera operates and is a good approximation of the modern
+photography process.
+
+The `PerspectiveMap()` command creates a singleton `Transformation` to perform
+the projective mapping. It can be applied individually, but is particularly
+powerful when composed with an `AffineMap` containing the position and
+orientation of the camera in your scene. For example, to transfer `points` in 3D
+space to 2D `screen_points` giving their projected locations on a virtual camera
+image, you might use the following code:
+
+```julia
+cam_transform = PerspectiveMap() ∘ inv(AffineMap(cam_rotation, cam_position))
```

Comment:
There's an extra bit to include in here. A camera projection matrix P (3x4) is normally formed as:
P = [K * R' | -R'*C0]
with R a rotation, C0 the camera pinhole, and K (3x3) the camera internals parameters.

I like having R and C0 described with the affine map as above; since after that part you're in a useful coordinate frame (I've been calling it camera frame)

So I think the PerspectiveMap should be PerspectiveMap(K) because its K that's defining the perspective part mapping points in R3 to the pencil of rays P2   (i.e. [x, y, z] -> [x', y', w].

Unless the perspective part is actually intersecting the pencil of rays with a plane when we homogenize the point in P2